src/main/java/au/org/ala/biocache/dto/DownloadDoiDTO.java

```diff
@@ -141,4 +151,17 @@ public void setAuthorisedRoles(List<String> authorisedRoles) {
     public void setApplicationMetadata(Map<String, String> applicationMetadata) {
         this.applicationMetadata = applicationMetadata;
     }
+
+    public String getDisplayTemplate(){
+        return  this.displayTemplate;
+    }
+
+    public void setDisplayTemplate(String displayTemplate){
+        if( validDisplayTemplates.contains(displayTemplate) )
```

Comment:
Use braces for this if condition to match the rest of the biocache-service code style.

Reply:
Don't think so. Found many instances of if statements without curly braces.

Reply:
If there are other exceptions to the style, they are exceptions that also need to be fixed. Please fix this instance.

Reply:
Agree with PA - ALA code style dictates we follow Oracle standard which is always use curly braces. Existence of code without braces is not a justification to commit code that doesn't follow ALA code style.